### PR TITLE
fixed the grid issue

### LINKED ISSUE #20: [OTHER] Content in the grid isn't responsive
### What would you like to share or ask?

gssoc

The content here overflows each grid container and isnt responsive for tablet and mobile view

Please allow me to work on this issue under GSSOC 2024.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Ankitapanda25**
## Changes proposed
the grid is made responsive according to the screen sizes
the text doesnt overflow anymore
the links of the URLs were made clickable
the links open in a separate tab

## Screenshots
[screen-capture (1).webm](https://github.com/AbhiPatel10/openzone/assets/123666302/1cd61a1e-6766-4bd4-8302-8604ab6fd26e)

## Note to reviewers
Please review my PR under GSSOC'24 and do give a feedback

### CODE REVIEW

**Comment by Ankitapanda25** [comment]
@AbhiPatel10 Please review my PR, it closes issue #20

File: src/styles/global.css [hunk 1]
```diff
@@ -80,4 +80,8 @@
     src: url('/src/fonts/DMMono-Regular.ttf') format('truetype');
     font-weight: normal;
     font-style: normal;
+}
+
+html{
```
Comment (AbhiPatel10) [inline src/styles/global.css:85]: Do not use overflow like this, If You facing overflow issue then find the reason of overflow and solve it there.

File: package-lock.json
Comment (AbhiPatel10) [inline package-lock.json:1]: Please Remove This File

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .yarn/install-state.gz
+/package-lock.json
```
Comment (AbhiPatel10) [inline .gitignore:8]: Thanks For adding this

**Review by AbhiPatel10 — APPROVED** [review]

**Pull request merged**